zio-http/jvm/src/main/scala/zio/http/netty/NettyRuntime.scala

```diff
@@ -16,15 +16,17 @@
 
 package zio.http.netty
 
+import scala.annotation.unused
+
 import zio._
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 
 import io.netty.channel._
 import io.netty.util.concurrent.{Future, GenericFutureListener}
 
-private[zio] trait NettyRuntime { self =>
+private[zio] final class NettyRuntime(zRuntime: Runtime[Any]) {
```

Comment:
I was struggling a bit to see how this could be beneficial as a trait, but I'm happy to revert this change if we think there's a reason for it